packages/back-end/src/services/features.ts

```diff
@@ -441,10 +470,26 @@ async function getFeatureDefinitionsResponse({
   features = scrubFeatures(features, capabilities);
   experiments = scrubExperiments(experiments, capabilities);
 
+  const includeAutoExperiments =
+    !!includeRedirectExperiments && !!includeVisualExperiments;
+
+  if (includeAutoExperiments) {
+    if (!includeRedirectExperiments) {
+      experiments = experiments.filter((e) =>
+        e.variations.some((v) => v.urlRedirect)
```

Comment:
This seems backwards.  Don't we want to return `false` in this filter call?  Also, there's the same issue of an array being used in a boolean check

Reply:
You're right. It looks like I got the two filters mixed up here. I added `changeType` to the `VisualChangesetInterface` like you recommended in a previous comment, but should we add `changeType` to `AutoExperiment` as well to avoid using the `variations` array for this filter?